### PR TITLE
docs: eks scaling

### PR DESCRIPTION
**Pull request by snkas**
Additional documentation on scaling node groups in EKS.

### CODE REVIEW

File: docs/enterprise/kubernetes-guides/eks/cluster.md [hunk 1]
```diff
@@ -58,14 +58,19 @@ metadata:
   region: us-west-1
   version: "1.30"
 
-# Node groups that can be used to scale the cluster up and down
+# Availability zones (AZ) (at least two zones must be specified)
+# Note: not all instance types are available in all availability zones
+availabilityZones: ["us-west-1a", "us-west-1c"]
```
Comment (lalithsuresh) [inline docs/enterprise/kubernetes-guides/eks/cluster.md:63]: I'm not sure I follow why we need two here?

File: docs/enterprise/kubernetes-guides/eks/scaling.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+   ```
+   Note: adding `spot: true` will enable the use of spot instances (see also
+   [documentation](https://eksctl.io/usage/spot-instances/)).
+
+2. Create the new node group:
+   ```
+   eksctl create nodegroup --config-file=eks-config.yaml
+   ```
+
+## Scaling
```
Comment (lalithsuresh) [inline docs/enterprise/kubernetes-guides/eks/scaling.md:39]: This is not scaling. Better to call this "changing instance types".

File: docs/enterprise/kubernetes-guides/eks/cluster.md (hunk 1 above)
Comment (snkas) [reply docs/enterprise/kubernetes-guides/eks/cluster.md:63]: eksctl command fails if there is less than two: https://docs.aws.amazon.com/eks/latest/userguide/network_reqs.html

> at least two subnets that are in different Availability Zones.

Comment (snkas) [reply docs/enterprise/kubernetes-guides/eks/cluster.md]: If I recall correctly, if they are not specified, eks picks two availability zones by itself, unaware of the availability zones specified in the node groups. If a node group specifies an availability zone which is not picked, eksctl will error.

Comment (lalithsuresh) [reply docs/enterprise/kubernetes-guides/eks/cluster.md]: Weird. I guess I never ran into it.

**Review by lalithsuresh — APPROVED** [review]

**Pull request closed without merging**